rebuildability_check.py

```diff
@@ -23,7 +23,7 @@ def result(self):
         description = "Temporary project including expanded copy of parent to verify if everything can be rebuild from scratch"
         rebuild_project = self.project.create_subproject("Rebuild", title, description) # how to handle if it exists? Clean it and use? What if we do not have permission to create subproject?
 
-        testing_packages = [package.copy(rebuild_project.name, expand=True) for package in packages]
+        testing_packages = [package.link(rebuild_project.name) for package in packages]
```

Comment:
note that only works for truely static targets. In case of update projects, you really want to have a revision to your link. Like osc linkpac --current does